src/wrapper.js

```diff
@@ -47,8 +45,7 @@ class Wrapper {
      * @return {Promise.<string>}
      */
     retrieveFromKMS(value){
-        //Wrap in promise to catch errors calling `this.storage`
-        return this.storage.getSecret({ name: value });
+        return this.storage ? this.storage.getSecret({ name: value }) : null;
```

Comment:
Should we return `Promise.resolve(null)` instead of just `null` here so we have a consistent return type. If this returns null and someone uses this as a promise it will throw an error when they try to call `.then()` or `.catch()` on it

Reply:
Ok I can do that.